packages/terra-date-picker/src/DatePickerField.jsx

```diff
@@ -222,13 +229,25 @@ const DatePickerField = (props) => {
     mergedInputAttrs = { 'aria-describedby': ariaDescriptionIds, ...inputAttributes };
   }
 
+  const helpLabel = help ? (
+    <div id="format" aria-label={`Date Format: ${intl.formatMessage({ id: 'Terra.datePicker.dateFormat' })}, ${help}`}>
+      {`(${intl.formatMessage({ id: 'Terra.datePicker.dateFormat' })})`}
```

Comment:
nitpick: `intl.formatMessage({ id: 'Terra.datePicker.dateFormat' })` appears in several places. Perhaps create a constant to store the string.

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1239/commits/80414a54b3cdc342639cdec83ffe2bb4b0126719